benchmarking/commons/benchmark_definitions/real_benchmark_definitions.py

```diff
@@ -27,4 +33,6 @@ def real_benchmark_definitions(
     return {
         "resnet_cifar10": resnet_cifar10_benchmark(sagemaker_backend, **kwargs),
         "lstm_wikitext2": lstm_wikitext2_benchmark(sagemaker_backend, **kwargs),
+        "mlp_fashionmnist": mlp_fashionmnist_benchmark(sagemaker_backend, **kwargs),
+        "distilbert_imdb": distilbert_imdb_benchmark(sagemaker_backend, **kwargs),
     }
```

Comment:
NIT: I am not sure if I understand how this relates to other changes in this PR (removal of outdated code). Could you explain?

Reply:
I am sometimes doing these things on the side. Just something I forgot to do in an earlier PR.